### PR TITLE
chore(repo): add CODEOWNERS

### PR DESCRIPTION
**Pull request by P0lip**

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,2 @@
+* @stoplightio/team-plaid
+/packages/rulesets/src/asyncapi  <add-asyncapi-team>
```
Comment (P0lip) [inline .github/CODEOWNERS]: @marbemac could you make sure to create some team and add https://github.com/magicmatatjahu?
@jonaslagoni would you like to be included as well?

Comment (magicmatatjahu) [reply .github/CODEOWNERS]: Sorry for such a delay! Thanks a lot! I feel honored :) If it would be possible @smoya himself was also interested.

Comment (jonaslagoni) [reply .github/CODEOWNERS]: @P0lip it would be an honor 🙂

Comment (smoya) [reply .github/CODEOWNERS]: >  If it would be possible @smoya himself was also interested.

It would be amazing! Very much appreciated if you do @P0lip !

**Review by marbemac — APPROVED** [review]

File: .github/CODEOWNERS (hunk 1 above)
Comment (P0lip) [reply .github/CODEOWNERS]: The invites have been sent. LMK in case you haven't received one.

**Pull request merged**